lib/launcher.js

```diff
@@ -38,6 +35,22 @@ class SeleniumStandaloneLauncher {
         }
     }
 
+    _getConfig (config) {
+        this.seleniumArgs = config.seleniumArgs || {}
+        this.seleniumInstallArgs = config.seleniumInstallArgs || {}
+        this.seleniumLogs = config.seleniumLogs
+        this.logToStdout = config.logToStdout
+
+        if (config.port) {
+            if (this.seleniumArgs.seleniumArgs === undefined) {
+                this.seleniumArgs.seleniumArgs = []
+            }
+            if (!this.seleniumArgs.seleniumArgs.includes('-port')) {
```

Comment:
let's not use `includes` because it is not supported in node lower than 6 I think